src/renderer/downloadsDropdown.tsx

```diff
@@ -40,6 +41,9 @@ class DownloadsDropdown extends React.PureComponent<Record<string, never>, State
             window.desktop.downloadsDropdown.focus();
         });
 
+        window.desktop.getVersion().then(({name}) => {
```

Comment:
I'm a bit surprised to see that this works when you're removing this field in the type definition below. I guess it gets its type from somewhere else?

Reply:
It does! It's here: https://github.com/mattermost/desktop/blob/2a88175b09e46fdae6b680d251825a711a1941f5/src/types/window.ts#L65